index.html

```diff
@@ -199,9 +199,9 @@ <h3>Downloads</h3>
                         <p><strong>Customize your download:</strong></p>
                         <ul>
                             <li>Set title</li>
-                            <li>Choose between video, audio or captions download</li>
-                            <li>Select the most suitable file format</li>
-                            <li>Download the resolution you need</li>
+                            <li>Video or audio only, with or without captions</li>
```

Comment:
Captions are a separate download option. Your phrasing makes it sound like they get downloaded alongside.
```suggestion
                            <li>Video, audio, or just the captions</li>
```